metadata-models/src/main/pegasus/com/linkedin/metadata/search/AggregationMetadata.pdl

```diff
@@ -22,5 +22,7 @@ record AggregationMetadata {
       value: string
       entity: optional Urn
       facetCount: long
+      // indicates that the FilterValue is part of the search request
```

Comment:
Broadly i very much dislike the entire FilterValue struct that has been just tossed in here (much earlier). But this feels like a useful way to capture that context, and even return it to the client if need be